<ui version="4.0" >
 <class>UIIrcMain</class>
 <widget class="QWidget" name="UIIrcMain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QTextEdit" name="teStatus" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>nickName@server:port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leNickServerPort" >
       <property name="text" >
        <string>userName@irc.freenode.net:6667</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbConnect" >
       <property name="text" >
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Channel name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leChannelName" >
       <property name="text" >
        <string>#monkeystudio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbJoin" >
       <property name="text" >
        <string>Join</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
